integration/internal/container/container.go

```diff
@@ -0,0 +1,48 @@
+package container
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/stretchr/testify/require"
+)
+
+type TestContainerConfig struct {
+	Name             string
```

Comment:
I wish we had this struct (and ops) for the actual `client` package! I guess we can look at doing that later

Reply:
That's definitely sthg we should do 👼 (It's now on my todo 😛)